src/components/resume-builder/components/edit-panel/EditPanelComponents.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { FaPlus } from 'react-icons/fa';
+
+interface InputFieldProps extends React.ComponentProps<'input'> {}
+export const InputField: React.FC<InputFieldProps> = ({
+  value,
+  onChange,
+  onBlur,
+  className,
+  ...otherInputProps
+}) => {
+  return (
+    <input
+      value={value}
+      onChange={onChange}
+      onBlur={onBlur}
+      className={`w-full rounded-md text-xs p-1 font-light border border-gray-300 ${className}`}
+      {...otherInputProps}
+    />
+  );
+};
+
+/**
+ * This interface and component were created to keep thr label and input field together, but currently i have decided to go without labels, hence this component is not being used anywhere. In future, it can be used anywhere that's why i am not removing it.
+ */
+interface LabelInputFieldPairProps extends React.ComponentProps<'input'> {
+  label: string;
+}
+export const LabelInputFieldPair: React.FC<LabelInputFieldPairProps> = ({
```

Comment:
If this is not being used anywhere. then it can be removed as afterwards also it won't be used again.